_data/people/meevans1.yml

```
@@ -0,0 +1,68 @@
+name: <Meirin Oan Evans>
+shortname: <meevans1>
+title: <HSF Training Coordinator>
+active: true
+institution: <University of Sussex>
+website: <https://meirinoanevans.wixsite.com/portfolio>
+photo: /assets/images/team/Meirin-Evans.jpg
+presentations:
+  - title: Introduction to Machine Learning
+    date: 2020-11-02
+    url: https://indico.cern.ch/event/958112/sessions/370197/#20201103
+    meeting: ML + GPU Training
+    meetingurl: https://indico.cern.ch/event/958112/
+    location: Online
+  - title: Teaching Machine Learning with ATLAS Open Data
+    date: 2020-10-23
+    url: https://indico.cern.ch/event/852553/contributions/4057691/attachments/2128826/3584684/TeachingMachineLearning.pdf
+    meeting: 4th Inter-experiment Machine Learning Workshop
+    meetingurl: https://indico.cern.ch/event/852553/
+    location: Online
+  - title: Introduction to Machine Learning
+    date: 2020-10-12
+    url: https://indico.ph.liv.ac.uk/event/103/contributions/1221/
+    meeting: STFC School on Data Intensive Science 2020
+    meetingurl: https://indico.ph.liv.ac.uk/event/103
+    location: Online
+  - title: Connecting teaching to research: ATLAS Open Data in Sussex courses
```

```suggestion
  - title: "Connecting teaching to research: ATLAS Open Data in Sussex courses"
```

Colon followed by space is not allowed in YAML. It makes a nested dict instead. (or something like that, it really just crashes in practice)